index.html

```diff
@@ -0,0 +1,217 @@
+  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+
+  <header class="header">
+    <a href="#" class="name">OSCAR</a>
+    <nav class="flex-nav">
+      <a href="#" class="toggleNav">Menu</a>
+      <ul class="open">
+        <li class="mhead"><a href="#">Portfolio</a></li>
+        <li class="mhead"><a href="#">About</a></li>
+        <li class="mhead"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hamburguer">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+    </nav>
+  </header>
+
+  <section class="wrapper">  
+    <div class="introduction">
+      <h1>I’m Alexandra<br>Glad to see you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <br>
+    </div>
+    <div class="letsconect"></div>
+    <div class="sicons"></div>  
+  </section>
+
+  <!-- Second Page -->
+  <section class="profile">
+    <ul class="exp">
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Portfolio.svg" alt="tannerbio">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Tonic</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+      
+<!-- Third Page -->
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Availability.png" alt="Availability">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Multi-Post Stories</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>  
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>  
+          </div>
```

Comment:
- [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
![image](https://user-images.githubusercontent.com/92303891/219876301-09d640e2-026a-441f-ad3f-748112f3e233.png)